tests/test_per_capita_consumption.py

```diff
@@ -1,19 +1,19 @@
 import ast
 import json
 from flask_api import status
-from models.pkv import PKV
+from models.pcc import PerCapitaConsumption
 from models.user import User, GenderType, StateType
 from tests.buzzn_test_case import BuzznTestCase
 from util.database import db
 from routes.per_capita_consumption import get_moving_average_annualized
-from tests.test_pkv_calculation import DAY_ZERO, DAY_ONE, PKV_DAY_ONE
+from tests.test_per_capita_consumption_calculation import DAY_ZERO, DAY_ONE, PCC_DAY_ONE
 
 
-class PKVTestCase(BuzznTestCase):
-    """ Unit tests for PKV route and functions(). """
+class PerCapitaConsumptionTestCase(BuzznTestCase):
+    """ Unit tests for per capita consumption route and functions(). """
```

Comment:
Please remove the brackets after "functions", I must have inserted them accidentally.